agent/recognition/binarymatch.py

```
@@ -0,0 +1,182 @@
+            # 因为 MAA 的 TemplateMatch 对白底黑线稿的兼容性通常更好
+            processed_img = np.full_like(img, 255) # 创建全白底图
+            processed_img[mask > 0] = [0, 0, 0]    # 将掩码区域（目标）涂黑
+
+            # 3. 调试输出
+            # -------------------------------------------------
+            if debug_mode:
+                import time
+                import os
+                debug_dir = "debug_images"
+                if not os.path.exists(debug_dir):
+                    try: os.makedirs(debug_dir)
```

Ruff (0.15.0)

[error] Multiple statements on one line (colon)

(E701)